00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/Content Comments.md

```diff
@@ -11,6 +11,8 @@ tags:
 
 Suggested standard:
 
+**Note:** *See the other files in this folder for progress on implementing this in each of the types of note.*
```

Comment:
I like these. And the status one as well. It makes this work a lot more transparent.
Just a small thing: I think it should be "types of note**s**"?